examples/cross_build/emscripten.rst

```diff
@@ -0,0 +1,256 @@
+.. _examples_cross_build_emscripten:
+
+Cross-building with Emscripten
+==============================
+
+This example demonstrates how to cross-build a simple C++ project using Emscripten and Conan.
+
+Conan supports building for both `asm.js <http://asmjs.org>`_ and `WASM
+<https://webassembly.org>`_, giving you the flexibility to target different
+JavaScript/WebAssembly runtimes in the browser.
+
+We recommend creating separate Conan profiles for each target. Below are
+recommended profiles and instructions on how to build with them.
+
+What’s the difference between asm.js and WASM?
+----------------------------------------------
+
+- **asm.js** is a subset of JavaScript optimized for speed. It is fully supported by all browsers (even older ones) and compiles to a large ``.js`` file.
+- **WebAssembly (WASM)** is a binary format that is smaller and faster to load and execute. Most modern browsers support it, and it is generally recommended for new projects. ``WASM`` is also easier to integrate with native browser APIs compared to ``asm.js``.
+
+Setting Up Conan Profiles
+-------------------------
+
+**For asm.js (JavaScript-based output):**
+
+.. code-block:: text
+
+   include(default)
+   [settings]
+   arch=asm.js
+   build_type=Release
+   os=Emscripten
+
+   [tool_requires]
```

Comment:
Maybe in the basic example would be better not to depend on a ConanCenter package such as ``emsdk``, but use the one installed by the user?. Then a second example can use the one from the package. 

Same use case as AndroidNDK in https://docs.conan.io/2/examples/cross_build/android/ndk.html, we can assume that devs building for webassembly will have their own emscripten installed in their machines and want to use it?